plugin/plugins/mijia/mijia_api/infrastructure/__init__.py

```
@@ -0,0 +1,35 @@
+"""基础设施层
+
+提供技术基础设施组件，如HTTP客户端、缓存、加密、日志等。
+
+模块：
+- http_client: HTTP客户端（同步和异步）
+- cache_manager: 缓存管理器
+- redis_client: Redis客户端（可选）
```

Ruff (0.15.7)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)